specs/disk_details_for_hardware_inventory.adoc

```diff
@@ -0,0 +1,178 @@
+= Specification for adding disk details in node hardware inventory
+
+Add disk details also as a part of node hardware inventory
+
+== Problem description
+
+Hardware inventory details are used to show exact status of the particular
+node. Currently disk details are missing in node hardware inventory. Add disk
+details in node hardware inventory and disk details should contain disk_id,
+device_name, fs_type, fs_uuid, model, mount_point, parent, size, name, disk_type,
+used, ssd, vendor, serial_no, firmware_version.
+
+== Use Cases
+
+Tendrl should show the current status of disk.
+
+== Proposed change
+
+* "hwinfo" and "lsblk" to find the disk details.
```

Comment:
I feel some of the details like classes etc should move to implementation section